vhdbuilder/packer/test/run-test.sh

```diff
@@ -40,15 +40,15 @@ else
 fi
 
 # Create the testing resource group
-az group create --name $TEST_VM_RESOURCE_GROUP_NAME --location ${AZURE_LOCATION} --tags "source=AgentBaker" "now=$(date +%s)" "branch=${GIT_BRANCH}"
+az group create --name "$TEST_VM_RESOURCE_GROUP_NAME" --location "${AZURE_LOCATION}" --tags "source=AgentBaker" "now=$(date +%s)" "branch=${GIT_BRANCH}"
```

Comment:
May as well put braces on the variable use as well? (Same for other cases further down.)

```suggestion
az group create --name "${TEST_VM_RESOURCE_GROUP_NAME}" --location "${AZURE_LOCATION}" --tags "source=AgentBaker" "now=$(date +%s)" "branch=${GIT_BRANCH}"
```

Reply:
Good idea, and I agree. However, I need to get this change into master and don't want to restart the build. Will do this if the build fails and I have to restart it tho :)